swagger.yaml

```diff
@@ -90,19 +90,19 @@ paths:
           description: "tweet will be created and posted"
           required: true
           schema:
-            $ref: "#/definitions/TweetsData"
```

Comment:
In your definition of `UserPTweetdata`:

```yaml
  UserPTweetdata:
    type: object
    properties:
      data:
        type: object
        properties:
          id:
            type: string
            example: 1234
          type:
            type: string
          attributes:
            $ref: "#/definitions/TweetsPData"
```

it will allow users to update their tweet ID which doesn't sound right. Also, can you give this definition a more descriptive name?

Take a look this example: https://github.com/osu-mist/slay-the-spire-cards-api/blob/master/swagger.yaml#L306-L315